MobileCenterCrashes/MobileCenterCrashes/MSCrashes.m

```diff
@@ -154,7 +154,15 @@ - (instancetype)init {
 - (void)applyEnabledState:(BOOL)isEnabled {
   [super applyEnabledState:isEnabled];
 
+  // Enabling
   if (isEnabled) {
+    
+    // PLCrashReporter keeps collecting crash reports even when the SDK is disabled,
+    // delete them only if current state is disabled.
+    if (!self.isEnabled){
+      [self.plCrashReporter purgePendingCrashReport];
```

Comment:
Is this actually disable (or stop) `PLCrashReporter`? Would be good to have a unit test for this scenario.